portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase

```diff
@@ -1232,7 +1232,7 @@ definition {
 	@description = "Verify that a Rich Text Field can be previewed"
 	@priority = "5"
 	test RichTextFieldCanBePreviewed {
-		property forms.functional = "false";
+		property forms.functional = "true";
```

Comment:
@wesleyroberts, is this change necessary for the test `TextFieldCanBePreviewed` pass?

Reply:
Changes  made @natocesarrego .